pip/cmdoptions.py

```diff
@@ -114,7 +114,7 @@ def getname(n):
     dest='quiet',
     action='count',
     default=0,
-    help='Give less output.')
+    help='Suppress all output except errors.')
```

Comment:
Also worth noting that (based on the referenced code) "Suppress all output except errors" is actually wrong - a single ```-q``` still shows WARNING level messages.